config/locales/de.yml

```diff
@@ -51,3 +51,8 @@ de:
         country: "Land"
         graduates_school_in: "Abitur vorraussichtlich in"
         already_graduated: "Bereits Abitur"
+  activerecord:
```

Comment:
Labeling look good, but I think it would be better to split the translation into different files (see merged PR #64) to prevent that the de.yml becomes to large and unreadable